docs/README-emscripten.md

```diff
@@ -35,42 +199,141 @@ all to make it happen.
 Please see the discussion at https://github.com/libsdl-org/SDL/issues/6385
 for some Javascript code to steal for this approach.
 
+SDL3 allows you to open the audio device at any time; it'll just throw away
+audio data until the user interacts with the page, and then start feeding
+the browser seamlessly, but for SDL2, you need to manage this yourself.
+
+
+## Rendering
+
+If you use SDL's 2D render API, it will use GLES2 internally, which
+Emscripten will turn into WebGL calls. You can also use OpenGL ES 2
+directly by creating a GL context and drawing into it.
+
+Calling SDL_RenderPresent (or SDL_GL_SwapWindow) will not actually
+present anything on the screen until your return from your mainloop
+function.
+
 
 ## Building SDL/emscripten
 
+First: do you _really_ need to build SDL from source?
+
+If you aren't developing SDL itself, have a desire to mess with its source
+code, or need something on the bleeding edge, don't build SDL. Just use
+Emscripten's packaged version!
+
+Compile and link your app with `-s USE_SDL=2` and it'll use a build of
+SDL packaged with Emscripten. This comes from the same source code and
+fixes the Emscripten project makes to SDL are generally merged into SDL's
+revision control, so often this is much easier for app developers.

```

Comment:
IDK if it's worth mentioning, but if you're using CMake you can just `find_package(SDL2)` and it hides the `-sUSE_SDL=2`. Oh and I _think_ dropping the space after `-s` is preferred now.